operator/internal/handlers/lokistack_create_or_update.go

```diff
@@ -148,6 +148,18 @@ func CreateOrUpdateLokiStack(
 		return nil, optErr
 	}
 
+	if opts.Stack.Size != lokiv1.SizeOneXDemo {
+		if opts.Stack.Template.Ingester.Replicas != 2 || opts.Stack.Replication.Factor != 2 {
+			if opts.Stack.Template.Ingester.Replicas <= opts.Stack.Replication.Factor {
+				return "", &status.DegradedError{
+					Message: "Invalid configuration: replication factor should be less than ingester replicas",
```

Comment:
Rewording it the other way around makes me feel users would be more likely to increase the number of ingester replicas
```suggestion
					Message: "Invalid configuration: ingester replicas should be bigger than the replication factor",
```

I also though if we want to use `fmt.Sprintf` to template the message with the observed values

Reply:
Good idea regarding the observed values!